test-app/tests/integration/components/input-field-test.gts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+import { fillIn, render, setupOnerror } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import InputField from '@crowdstrike/ember-toucan-core/components/form/input-field';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+
+module('Integration | Component | InputField', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      <InputField
+        @label="Label"
+        type="text"
+        data-input
+        />
+    </template>);
+
+    const label = '[data-label]'
+    const input = '[data-input]';
+
+    assert.dom(label).exists('Expected to have label block rendered');
+    assert.dom(label).hasText('Label', 'Expected to have label text "label"');
+
+    assert.dom(input).exists('Expected to have input tag rendered');
+    assert.dom(input).hasAttribute('type', 'text');
```

Comment:
We may want to verify the `id` gets set here as well:

```js
assert.dom('[data-input]').hasAttribute('id');
```

Reply:
okay I think this is here, but you might want to confirm since there is no "Outdated" label

Reply:
Okay yeah it's here: https://github.com/CrowdStrike/ember-toucan-core/pull/35/files#diff-b1d74a6d42d39cd9e5f8eb62146064c2694715629721f598278442b2a9249855R31